### ISSUE TITLE
JAWS does no count the first item of dopdowns

### ISSUE THREAD

**Issue by Heather-Qian** [FreedomScientific/VFO-standards-support#711]
### Summary

> When entering a dropdown/combobox, JAWS does not announce the number of the first item or default item like "1 of 4". The counting starts from the second item. Whenever moving focus to the first item, Jaws announces only it as “to change for selection use arrow keys. “ However, we tested with NVDA that it can announce the number.
> 

### Expected result

> We were expecting counting to get started from the first list item. 

### Actual result

>  Currently counting of list item starts from the 2nd element. 

### Example

> See the video.
[counting from second element_.webm](https://user-images.githubusercontent.com/82977101/231811422-b738b897-77a3-4d56-9551-4d54910c56a6.webm)

#### JAWS version and build number

•	JAWS 2022.2207.25 ILM
 
#### Operating System and version

•	Windows 10

#### Browser and version:

•	Chrome 112.0.5615.49
